nats/testdata/nats-tls.conf

```diff
@@ -0,0 +1,10 @@
+# Simple TLS config file
+
+port: 4443
+net: 0.0.0.0 # net interface
+
+tls {
+  cert_file:  "/tls/redis.crt"
+  key_file:   "/tls/redis.key"
```

Comment:
`ca_file` is missing here: https://docs.nats.io/running-a-nats-service/configuration/securing_nats/tls

Reply:
During the CI, the file will be `/tls/ca.crt`